doc/docgen.rst

```diff
@@ -229,6 +229,146 @@ Output::
 Note that the `jsondoc`:option: command outputs its JSON without pretty-printing it,
 while `jsondoc0`:option: outputs pretty-printed JSON.
 
+
+Referencing Nim symbols: simple documentation links
+===================================================
+
+You can reference Nim identifiers from Nim documentation comments (currently
+only inside one ``.nim`` file).
```

Comment:
maybe also mention that it works for nim files including an rst file?

Reply:
ok, mentioned.